mnq/sns-instances-notification-system/README.md

```diff
@@ -0,0 +1,40 @@
+# SNS Instances Notification System
+
+This repository contains the source code for the this tutorial: [Creating a notification system with Scaleway SNS and Instances.](https://github.com/scaleway/docs-content/blob/int-add-mnq-tuto/tutorials/sns-instances-notification-system/index.mdx)
+
+## Requirements
+
+This example assumes that you are familiar with:
+
+- how System Notification Service work. You can find out more in the [SNS Scaleway official documentation](https://www.scaleway.com/en/docs/serverless/messaging/quickstart/)
+- how Instances work. If needed, you can visit the [Instances Quickstart documentation](https://www.scaleway.com/en/docs/compute/instances/quickstart/).
+- how to create a SSH key. If not done already, you can follow the first 6 steps of [this walkthrough](https://www.scaleway.com/en/docs/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/#how-to-upload-the-public-ssh-key-to-the-scaleway-interface).
+
+## Context
+
+This example shows you how to set up a Notification System with Terraform across Instances using Scaleway's products, through a simulated CPU monitoring example.
+
+## Setup
+
+Once you have cloned this repository you can run:
+
+```console
+terraform init
+terraform plan
+terraform apply
+```
+You will be prompted to enter your public ssh. You will find them in the `~/.ssh/` folder:
+
+```console
+cat ~/.ssh/*.pub
+```
+
+## Testing
+
+Connect to the subscriber Instance with this URL ```http://<your_subscriber_ip_address>:8081``` (you can find the address on the Console Instances page)
```

Comment:
```suggestion
- Connect to the subscriber Instance with this URL ```http://<your_subscriber_ip_address>:8081``` (you can find the address on the Console Instances page)
```

**issue(format):** this looks like a list so should have hyphens/asterisks before each item